apps/avifenc.c

```diff
@@ -827,6 +858,12 @@ int main(int argc, char * argv[])
         }
         // Don't subsample when using AVIF_MATRIX_COEFFICIENTS_IDENTITY.
         input.requestedFormat = AVIF_PIXEL_FORMAT_YUV444;
+        // Quality.
+        if ((quality >= 0 && quality != AVIF_QUALITY_LOSSLESS) || (qualityAlpha >= 0 && qualityAlpha != AVIF_QUALITY_LOSSLESS)) {
+            fprintf(stderr, "Quality cannot be set in lossless mode, except to 100.\n");
```

Comment:
replace 100 with %d and AVIF_QUALITY_LOSSLESS

Reply:
Done.